Numbers/unit_converter.py

```diff
@@ -2,27 +2,54 @@
 # 21EUR, 22UAH, 23USD ----> RUB
 # 31kg -> pound, 32pound -> kg
 
-def unit_converter(convert: int, value: int) -> str:
-    if convert == 11:
-        return f"{value} °C is equal to {(value*1.8)+32} °F"
-    elif convert == 12:
-        return f"{value} °F is equal to {(value-32)*0.5556} °C"
-    elif convert == 21:
-        return f"{value} EUR is equal to {value*86.93} RUB"
-    elif convert == 22:
-        return f"{value} UAH is equal to {value*2.72} RUB"
-    elif convert == 23:
-        return f"{value} USD is equal to {value*77.37} RUB"
-    elif convert == 31:
-        return f"{value} Pound is equal to {value*0.453592} kg"
-    elif convert == 32:
-        return f"{value} kg is equal to {value*2.2046} Pound"
-    else:
-        return f"I don't understand"
-
-
-print(unit_converter(11,12))
+def convert_unit(frm: str, to: str, value: int) -> str:
```

Comment:
It's hugely better to return a numeric value for a converted unit. And make a separate method to print conversion results.

Then this method could be unit tested properly.

Also, it should be float here.

Reply:
method to print conversion.
I don't get it too much. can I have an example, please?

Reply:
Maybe don't even print. It's not necessary here.
Just remove all prints here.

Suppose that you want to use this code somewhere else and you need to store data in a database. Then it will be better to return just a numeric value.

**long story short:** remove all prints here, return values anly